lib/rails_multitenant/multitenant_model.rb

```diff
@@ -18,7 +18,9 @@ def multitenant_on(context_entity_id_field, required: true)
         scope_sym = "from_current_#{context_entity}".to_sym
 
         scope scope_sym, -> do
-          where(context_entity_id_field => GlobalContextRegistry[context_entity_id_field])
+          unless GlobalContextRegistry[:use_unscoped_queries]
```

Comment:
Should we add `GlobalContextRegistry#use_unscoped_queries?` method to encapsulate this data structure a bit more?